### ISSUE TITLE
Reindex all exhibits in production

### ISSUE THREAD

**Issue by drh-stanford** [sul-dlss/exhibits#776]
There are a couple fixes -- Mirador drag-n-drop and IIIF badges -- that require reindexing, so we need to work this process out.

**Comment by camillevilla** [comment]
Implicated tickets: 

- #738 / #741 Bug: `Bibliography` displays on all exhibits
- #745 / #766 Bug: IIIF logo showing on all SDR objects
- #710 Bug: IIIF drag-n-drop (caused by `manifest.json` endings)

**Comment by drh-stanford** [comment]
Nice comment :). The bibliography display problem (#738) is a JavaScript fix that won't require reindexing. The custom tags disappearing (#753) is a side-effect of reindexing so it won't required reindexing.

#768 is also implicated as it's a problem with the ReindexJob not running in production. This needs to be fixed before we can reindex any exhibit in prod (stage seems to be fine -- I'm trying to reproduce the problem in stage).

**Comment by drh-stanford** [comment]
We can reindex all of the exhibits via a single task `rake spotlight:reindex` -- see https://github.com/projectblacklight/spotlight/blob/master/lib/tasks/spotlight_tasks.rake#L128

**Comment by drh-stanford** [comment]
This is blocked by #768. We need to reindex all exhibits in production (using `rake spotlight:reindex`) so that #710 and #745 can be fixed -- they are index-time solutions.

**Comment by drh-stanford** [comment]
#745 has a new PR ( #794) that does not depend on indexing, but the current fix for #710 does (see https://github.com/sul-dlss/exhibits/blob/master/app/models/spotlight/dor/indexer.rb#L48). We could implement a non-index-time fix for #710 if needed (e.g., strip `.json` from all manifeset URLs at display time). cc @mejackreed

**Comment by jkeck** [comment]
It appears (after a little more whack-a-mole with some data issues) that most exhibits have been or are in the process of re-indexing.

There are a few issues that are causing some re-indexes not to complete properly:

1. One of the issues is trying to parse a Welcome manifest in the `team-test` exhibit.  The issue is the welcome manifest has multiple services and the iiif-presentation gem does not handle that yet (see: iiif-prezi/osullivan#68).
2. The other exhibit that is failing (`test-cb-20161213`) is getting 400's from solr repeatedly.  I haven't dug too deep into that one, but I'm wondering if we can just delete (cc: @cbeer)
